.htaccess

```diff
@@ -110,13 +110,13 @@ RewriteRule "^keyboards/(?!index\.php|install|keyboard|session|share)([^/]+)$" "
 RewriteRule "^keyboards/languages/(.*)" "/keyboards/index.php?q=l:id:$1" [L]
 
 # /keyboards/download to /keyboards/download.php
-RewriteRule "^keyboards/download(.php)?" "/keyboards/download.php" [L]
+RewriteRule "^keyboards/download(.php)?" "/keyboards/download.php" [L,QSA]
```

Comment:
`L,QSA` and not `END,QSA`?

Reply:
It probably is [END,QSA]. I'll change in follow-on